src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -3,18 +3,20 @@
 import lotto.dto.LottoNumberDto;
 
 import java.util.*;
-import java.util.stream.Collectors;
 
 public class LottoNumber implements Comparable<LottoNumber> {
 
     private static final int LOTTO_MIN_NUMBER = 1;
     private static final int LOTTO_MAX_NUMBER = 45;
     private static Map<Integer,LottoNumber> lottoNumbers = new HashMap<>();
     private int lottoNumber;
+    private static List<LottoNumber> lottoNumberList = new ArrayList<>();
```

Comment:
자바 컨벤션 중 클래스 구성요소 정렬에 대한 내용이 있어요.

```
1. 상수
2. 필드
3. 생성자
4. 메소드
```

상수는 필드 위에 위치시켜주세요 :)

```suggestion
    private static final List<LottoNumber> lottoNumberList = new ArrayList<>();
  
    private int lottoNumber;
```

한 칸 정도 띄워주시면 보기에 더 좋습니다.
그리고 Map과 List는 final로 고정시켜 주시는게 더 좋을 것 같습니다.
참조가 변하진 않을거라고 판단되거든요.

Reply:
상수로 선언하고 다시 작성하였습니다..!
상수랑 필드사이에 한 줄 띄우니 더 읽기 좋아진 것 같습니다!